### PR TITLE
[stereocam] uartrotation->state2camera, streamline naming stereo_cam,…

### PR DESCRIPTION
**Pull request by dewagter**
- all related files in stereocam directory
 - uartrotation->state2camera
 - stereo_cam -> stereocam
 - stereo -> stereocam
 - stereoavoid = the avoid version that plans around obstacles during a nav_line auto2 flight -> nav_line_avoid
 - avoid with random exploration -> stereocam_droplet

### CODE REVIEW

**Comment by dewagter** [comment]
- [ ] build server checks
 - [ ] inform involved users

File: sw/airborne/modules/stereocam/droplet/stereocam_droplet.c [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    case 99:     // Turn
+      heading += 4;
+      if (heading > 360) { heading = 0; }
+      nav_set_heading_rad(RadOfDeg(heading));
+      break;
+    default:    // do nothing
+      break;
+  }
+
+#if STEREO_LED
```
Comment (flixr) [inline sw/airborne/modules/stereocam/droplet/stereocam_droplet.c]: Should be `#ifdef`

**Comment by flixr** [comment]
Like the cleanup, thx!

**Comment by flixr** [comment]
For what build server checks are you waiting there?
All confs are only tested nightly on the master branch....

**Comment by dewagter** [comment]
@rmeertens 
@kirkscheper 

Please note your modules were renamed. All stereocam modules are in the stereocam module folder and module names are grouped with the stereocam_ prefix.

All on-autopilot-board (linux-only) computer vision modules are grouped with prefix cv_

**Comment by rmeertens** [comment]
Thanks for the warning. Love the changes!

**Pull request merged**